cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -132,6 +136,12 @@ if [[ $SNLSYSTEM == "cts1" ]] ; then
   systemNameTypeMatchedListHostNames[cts1]=$SNLCLUSTER
 fi
 
+# TODO: vortex doesn't set SNLSYSTEM -- how do we tell if we're on an ats2 system?
```

Comment:
Yes, this is the only way that I know of.  Therefore, to support other ATS-2 systems at LLNL like 'lassen' and 'rzansel', we will have to put in other similar hostname greps as well.